src/extension/sldService/src/main/java/org/geoserver/sldservice/utils/classifier/RasterSymbolizerBuilder.java

```diff
@@ -370,7 +424,7 @@ public ColorMap createCustomColorMap(
             breaks[i + 1] = Converters.convert(classifier.getMax(i), Double.class);
         }
 
-        return getColorMapFromBreaks(breaks, open, continuous);
+        return getColorMapFromBreaks(breaks, open, continuous, null);
```

Comment:
If instructed to build percentages while doing a custom color map, they should show up. It's possible to calculate them using a histogram operation.

Reply:
fixed